elasticai/creator/vhdl/generator/precomputed_scalar_function.py

```diff
@@ -82,17 +77,24 @@ def __call__(self) -> list[str]:
             "x : in signed(DATA_WIDTH-1 downto 0)",
             "y : out signed(DATA_WIDTH-1 downto 0)",
         ]
-        code = "\n".join(chain([get_libraries_string()], chain(entity()), [""]))
-        code += get_architecture_header_string(
-            architecture_name=self.architecture_name, component_name=self.component_name
-        )
-        code += get_process_string(
-            component_name=self.component_name,
+        library = Library()
+        code = "\n".join(chain(chain(library()), chain(entity()), [""]))
```

Comment:
fyi: you can use `chain.from_iterable([library(), entity(), [""]])` to simplify this line a little bit